src/pages/settings/Payments/AddPayPalMePage.js

```diff
@@ -22,107 +22,90 @@ import Icon from '../../../components/Icon';
      */
-    setPayPalMeUsername() {
-        const isValid = ValidationUtils.isValidPaypalUsername(this.state.payPalMeUsername);
-        if (!isValid) {
-            this.setState({payPalMeUsernameError: true});
+    const setPayPalMeData = () => {
+        if (!ValidationUtils.isValidPaypalUsername(payPalMeUsername)) {
+            setPayPalMeUsernameError(true);
             return;
         }
-        this.setState({payPalMeUsernameError: false});
-        User.addPaypalMeAddress(this.state.payPalMeUsername);
+        setPayPalMeUsernameError(false);
+        User.addPaypalMeAddress(payPalMeUsername);
 
-        Growl.show(this.props.translate('addPayPalMePage.growlMessageOnSave'), CONST.GROWL.SUCCESS, 3000);
+        Growl.show(props.translate('addPayPalMePage.growlMessageOnSave'), CONST.GROWL.SUCCESS, 3000);
         Navigation.navigate(ROUTES.SETTINGS_PAYMENTS);
-    }
-
-    focusPayPalMeInput() {
-        if (!this.payPalMeInputRef) {
-            return;
-        }
-
-        this.payPalMeInputRef.focus();
-    }
+    };
 
-    render() {
-        return (
-            <ScreenWrapper onEntryTransitionEnd={this.focusPayPalMeInput}>
-                <HeaderWithCloseButton
-                    title={this.props.translate('common.payPalMe')}
-                    shouldShowBackButton
-                    onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS_PAYMENTS)}
-                    onCloseButtonPress={() => Navigation.dismissModal(true)}
-                />
-                <View style={[styles.flex1, styles.p5]}>
-                    <View style={[styles.flex1]}>
-                        <Text style={[styles.mb4]}>
-                            {this.props.translate('addPayPalMePage.enterYourUsernameToGetPaidViaPayPal')}
+    return (
+        <ScreenWrapper onEntryTransitionEnd={() => payPalMeInput.current && payPalMeInput.current.focus()}>
+            <HeaderWithCloseButton
+                title={props.translate('common.payPalMe')}
+                shouldShowBackButton
+                onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS_PAYMENTS)}
+                onCloseButtonPress={() => Navigation.dismissModal(true)}
+            />
+            <View style={[styles.flex1, styles.p5]}>
+                <View style={[styles.flex1]}>
+                    <Text style={[styles.mb4]}>
+                        {props.translate('addPayPalMePage.enterYourUsernameToGetPaidViaPayPal')}
+                    </Text>
+                    <TextInput
+                        ref={payPalMeInput}
+                        label={props.translate('addPayPalMePage.payPalMe')}
+                        autoCompleteType="off"
+                        autoCorrect={false}
+                        value={payPalMeUsername}
+                        placeholder={props.translate('addPayPalMePage.yourPayPalUsername')}
+                        onChangeText={(text) => { setPayPalMeUsername(text); setPayPalMeUsernameError(false); }}
```

Comment:
NAB, feels a bit more common to see these on separate lines (though technically perfectly fine code-wise)
```suggestion
                        onChangeText={(text) => { 
                            setPayPalMeUsername(text); 
                            setPayPalMeUsernameError(false); 
                        }}
```